### ISSUE TITLE
Possible to use Library on div class whitout data-attribute

### ISSUE THREAD

**Issue by melt-cdk** [mciastek/sal#102]
Hello,

it's possible to use sal.js on class div ? I can't use data-attribute ?

tx

**Comment by mciastek** [comment]
Yes, it's possible. You need to set `selector` (see [Advanced options](https://github.com/mciastek/sal#advanced-options)) value to some valid selector in options object passed to `sal()`. Then you need to define you own animations in CSS. You can use https://github.com/mciastek/sal/blob/master/src/sal.scss as reference.

**Comment by terry-pixel** [comment]
thank you, do you have an example to help?

**Comment by gingerchew** [comment]
Like this: 

```js
sal({
    selector: '.my-selector'
})
```
